src/scenes/OrganizationProjectView.re

```diff
@@ -29,36 +29,209 @@ module Styles = {
+            )
+          }
+          onSubmitFail={errors =>
+            switch (Belt.List.head(errors)) {
+            | Some((_field, error)) =>
+              snackbar.send(
+                Set(Open(Belt.Option.getWithDefault(error, ""))),
+              )
+            | None =>
+              snackbar.send(
+                Set(
+                  Open({j|Não foi possível processar essa alteração|j}),
+                ),
+              )
+            }
+          }
+          initialState={name: ""}
+          i18n=ReForm.Validation.I18n.ptBR
+          schema=[(`name, Required)]>
+          ...{({handleChange, form}) =>
+            <View style=Styles.wrapper>
+              <AppBar
+                title="Project"
+                renderLeft={_ => <BackButton onPress={_ => closeModal()} />}
+                renderRight={_ =>
+                  <DeleteButton
+                    onPress={() =>
+                      Js.Promise.(
+                        removeOrganizationProject(
+                          ~variables=
+                            RemoveOrganizationProjectMutation.make(
+                              ~input={
+                                "organizationId": organizationId,
+                                "projectId": projectId,
+                              },
+                              (),
+                            )##variables,
+                          (),
+                        )
+                        |> then_(result =>
+                             RemoveOrganizationProjectMutationContainer.convertJsInputToReason(
+                               result,
+                             ).
+                               data
+                             ->Belt.Option.map(data =>
+                                 data##removeOrganizationProject
+                               )
+                             ->Belt.Option.map(removeOrganizationProject =>
+                                 switch (removeOrganizationProject##error) {
+                                 | Some(error) =>
+                                   snackbar.send(Set(Open(error)));
+                                   Js.Promise.resolve();
+                                 | _ =>
+                                   snackbar.send(
+                                     Set(
+                                       Open(
+                                         {j|Infelizmente, nosso servidor não conseguiu processar essa requisição. Por favor, tente novamente.|j},
+                                       ),
+                                     ),
+                                   );
+                                   Js.Promise.resolve();
+                                 }
+                               )
+                             ->Belt.Option.getWithDefault(
+                                 Js.Promise.resolve(),
+                               )
+                           )
+                        |> then_(() =>
```

Comment:
with this here no matter what it will always log "Project Deletado", the handling should be top in the previous `then_`

Reply:
improved current logic there, please take a look later